MimeKit/Header.cs

```diff
@@ -1330,14 +1330,16 @@ public static string Unfold (string text)
 			}
 
 			int count = endIndex - startIndex;
-			char[] chars = new char[count];
+			Span<char> chars = count < 16
+				? stackalloc char[16]
+				: new char[count];
 
```

Comment:
Does using Span<> here really help?

Reply:
Okay, so if my quick research is correct (don't have time for a full reading because I'm about to run out), I guess using Span<> with stackalloc removes the need to declare the method `unsafe`?

Reply:
This is always a WIN when in throughput on .NET 5+ [we eliminate an allocation and clearing the stack], but there's a small penalty on older runtimes where we don't have SkipLocalsInit. This (may) be made up for avoiding the allocation in throughput.